.circleci/verbatim-sources/workflows-docker-builder.yml

```diff
@@ -1,11 +1,4 @@
   docker_build:
-    triggers:
```

Comment:
These changes need to be restored before merging.

Reply:
Yep, I was trying to switch to a python 3.8 image, will add them back later